app/views/layout.erb

```diff
@@ -59,6 +59,9 @@
                           <a href="#friends" class="smoothScroll">Our Friends</a>
```

Comment:
This div doesn't exist in code, I don't think we plan to implement this either, please remove if that is the case.